From b0adeee823a32699f2e80cb78fc679028354190d Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Thu, 20 Dec 2012 10:53:06 -0800
Subject: [PATCH] clk: zynq: Log PLL bypass/enable

Enabling and bypassing PLLs is rather important. Therefore a
kernel info message is issued on those events.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
---
 drivers/clk/zynq/pll.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/drivers/clk/zynq/pll.c b/drivers/clk/zynq/pll.c
index 8b889ea..bb5d287 100644
--- a/drivers/clk/zynq/pll.c
+++ b/drivers/clk/zynq/pll.c
@@ -292,6 +292,8 @@ static int zynq_pll_enable(struct clk_hw *hw)
 	if (!clk->bypassed)
 		return 0;
 
+	pr_info("PLL: Enable\n");
+
 	/* Power up PLL and wait for lock before removing bypass */
 	spin_lock_irqsave(clk->lock, flags);
 
@@ -326,6 +328,8 @@ static void zynq_pll_disable(struct clk_hw *hw)
 	if (clk->bypassed)
 		return;
 
+	pr_info("PLL: Bypass\n");
+
 	/* Set bypass bit and shut down PLL */
 	spin_lock_irqsave(clk->lock, flags);
 
-- 
1.7.1

